espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Model/AppiumStatus.java

```diff
@@ -5,32 +5,34 @@
+    ELEMENT_NOT_VISIBLE(11, "An element command could not be completed because the element is not visible on the page"),
+    INVALID_ELEMENT_STATE(12, "An element command could not be completed because the element is in an invalid state (error.g. attempting to click a disabled element)"),
+    UNKNOWN_ERROR(13, "An unknown server-side error occurred while processing the command"),
+    ELEMENT_IS_NOT_SELECTABLE(15, "An attempt was made to select an element that cannot be selected"),
+    JAVASCRIPT_ERROR(17, "An error occurred while executing user supplied JavaScript"),
+    XPATH_LOOKUP_ERROR(19, "An error occurred while searching for an element by XPath"),
+    TIMEOUT(21, "An operation did not complete before its timeout expired"),
+    NO_SUCH_WINDOW(23, "A request to switch to a different window could not be satisfied because the window could not be found"),
+    INVALID_COOKIE_DOMAIN(24, "An illegal attempt was made to set a cookie under a different domain than the current page"),
+    UNABLE_TO_SET_COOKIE(25, "A request to set a cookie's value could not be satisfied"),
     UNEXPECTED_ALERT_OPEN(26, "A modal dialog was open, blocking this operation"),
-    NO_ALERT_OPEN_ERROR(27, "An attempt was made to operate on a modal dialog when one was not open."),
-    SCRIPT_TIMEOUT(28, "A script did not complete before its timeout expired."),
-    INVALID_ELEMENT_COORDINATES(29, "The coordinates provided to an interactions operation are invalid."),
-    IME_NOT_AVAILABLE(30, "IME was not available."),
-    IME_ENGINE_ACTIVATION_FAILED(31, "An IME engine could not be started."),
-    INVALID_SELECTOR(32, "Argument was an invalid selector (error.g. XPath/CSS)."),
-    SESSION_NOT_CREATED_EXCEPTION(33, "A new session could not be created."),
-    MOVE_TARGET_OUT_OF_BOUNDS(34, "Target provided for a move action is out of bounds."),
-    JSON_DECODER_ERROR(35, "Could not decode action/params of command, please check format!");
+    NO_ALERT_OPEN_ERROR(27, "An attempt was made to operate on a modal dialog when one was not open"),
+    SCRIPT_TIMEOUT(28, "A script did not complete before its timeout expired"),
+    INVALID_ELEMENT_COORDINATES(29, "The coordinates provided to an interactions operation are invalid"),
+    IME_NOT_AVAILABLE(30, "IME was not available"),
+    IME_ENGINE_ACTIVATION_FAILED(31, "An IME engine could not be started"),
+    INVALID_SELECTOR(32, "Argument was an invalid selector (error.g. XPath/CSS)"),
+    SESSION_NOT_CREATED_EXCEPTION(33, "A new session could not be created"),
+    MOVE_TARGET_OUT_OF_BOUNDS(34, "Target provided for a move action is out of bounds"),
+    JSON_DECODER_ERROR(35, "Could not decode action/params of command, please check format!"),
+    INVALID_CONTEXT_ERROR(36, "The operation could not be performed in the current context"),
+    BAD_PARAMETERS_ERROR(37, "The parameters specified for the operation are incorrect");
```

Comment:
(these numbers are defined by the JSONWP spec so we can't just make up new ones)